django/middleware/cache.py

```diff
@@ -144,6 +147,14 @@ def process_request(self, request):
             request._cache_update_cache = True
             return None  # No cache information available, need to rebuild.
 
+        # Setting cache Age
```

Comment:
Could you expand the comment to describe the calculation a little bit more? The logic isn't obvious to me.